src/FilterWrapper/components/FilterActionsPopover/FilterActionsPopover.tsx

```diff
@@ -1,102 +1,84 @@
+	filterActions,
+	filterActionsCaption
+}) => {
+	const [isOpen, setIsOpen] = React.useState(false);
 
-	public state = {
-		isOpen: false
+	const handleClickFilter = () => {
+		setIsOpen(!isOpen);
 	};
 
-	public handleClickFilter = () => {
-		this.setState(state => ({ isOpen: !state.isOpen }));
+	const hideListPopover = () => {
+		setIsOpen(false);
 	};
 
-	public hideListPopover = () => {
-		this.setState(state => ({ ...state, isOpen: false }));
-	}
-
-	public render() {
-		const { filterActions, filterActionsCaption } = this.props;
+	return filterActions && filterActions.length ? (
+		<div
+			className={cn("kit-filter-actions-popover", IsntNeutralZoneMarker)}
+		>
+			<span
+				className="kit-filter-actions-popover__title"
+				onClick={handleClickFilter}
+			>
+				{filterActionsCaption}
+			</span>
+			<ul
+				className={cn("kit-filter-actions-popover__list", {
+					"kit-filter-actions-popover__list_hide": !isOpen
+				})}
+			>
+				{filterActions.map((item, index) => {
+					if (isFilterAction(item)) {
+						const { key, onClick, name } = item as FilterAction;
 
-		const { isOpen } = this.state;
+						const handleClickItem = () => {
+							hideListPopover();
+							onClick();
+						};
 
-		return filterActions && filterActions.length ? (
-			<div className={cn(
-				"kit-filter-actions-popover",
-				IsntNeutralZoneMarker
-			)}>
-				<span
-					ref={this.refElement}
-					className="kit-filter-actions-popover__title"
-					onClick={this.handleClickFilter}
-				>
-					{filterActionsCaption}
-				</span>
-				<OverflowVisibleContainer parentRef={this.refElement} onNeutralZoneClick={this.hideListPopover}>
-					<ul className={
-						cn("kit-filter-actions-popover__list", {
-							"kit-filter-actions-popover__list_hide": !isOpen
-						})
-					}>
-						{
-							filterActions.map((item, index) => {
-								if (isFilterAction(item)) {
-									const { key, onClick, name } = item as FilterAction;
-									return (
-										<li
-											className="kit-filter-actions-popover__item"
-											key={key}
-											onClick={() => {
-												this.hideListPopover();
-												onClick();
-											}}
-										>
-											{name}
-										</li>
-									);
-								}
-								return (
-									<li
-										className="kit-filter-actions-popover__item"
-										key={`kit-filter-actions-popover__item-key-${index}`}
-										onClick={this.hideListPopover}
-									>
-										{item}
-									</li>
-								);
-							})
-						}
-					</ul>
-				</OverflowVisibleContainer>
-			</div>
-		) : null;
-	}
-}
+						return (
+							<li
+								className="kit-filter-actions-popover__item"
+								key={key}
+								onClick={handleClickItem}
+							>
+								{name}
+							</li>
+						);
+					}
+					return (
+						<li
+							className="kit-filter-actions-popover__item"
+							key={`kit-filter-actions-popover__item-key-${index}`}
```

Comment:
Можно просто `index`.
```suggestion
							key={index}
```